src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -46,6 +46,22 @@ class ActivityFeed extends React.Component<Props, State> {
 
     feedContainer: null | HTMLElement;
 
+    componentDidMount() {
+        if (this.feedContainer) {
+            this.feedContainer.scrollTop = this.feedContainer.scrollHeight;
+        }
+    }
+
+    componentDidUpdate(prevProps) {
+        if (
+            this.feedContainer &&
+            prevProps.feedItems === undefined &&
+            this.props.feedItems
+        ) {
+            this.feedContainer.scrollTop = this.feedContainer.scrollHeight;
```

Comment:
What if a comment is edited? Then we would jump to the bottom when the props change?

Reply:
I think this looks ok. Its checking if the feedItems go from undefined to a truthy value

Reply:
What if the length changes due to a new comment being added? Does that need special handling?

Reply:
When a new comment is added, the onSubmit prop in ApprovalCommentForm will automatically scroll the div to the bottom